test/project-service.ts

```diff
@@ -64,7 +64,7 @@ class ProjectIntegrationTest {
 			var projectDir = path.join(tempFolder, projectName);
 			var appDirectoryPath = path.join(projectDir, "app");
 			var platformsDirectoryPath = path.join(projectDir, "platforms");
-			var tnsProjectFilePath = path.join(projectDir, ".tnsproject");
+			var tnsProjectFilePath = path.join(projectDir, "package.json");
```

Comment:
you can use `let` instead of `var`. Also I think the name of the variable should be changed to projectFilePath or packageJson for example.